FEM/rf_st_new.cpp

```diff
@@ -3892,12 +3883,39 @@ void CSourceTermGroup::SetSurfaceNodeValueVector(CSourceTerm* st, Surface* m_sfc
 	}
 }
 
+#if defined(USE_PETSC) // || defined (other parallel linear solver lib). //WW. 06.2016
+void CSourceTermGroup::removeGhostElements(std::vector<long>& node_ids,
```

Comment:
Here, 'element' means the element of source terms. The name has been changed to ``removeGhostSourceTerms``